pages/neighbourhoods.py

```diff
@@ -0,0 +1,434 @@
+df['%_ZVWKFARMACIE_user'] = df['%_ZVWKFARMACIE_user'].mask( (df['YEAR'] >2020), np.nan)
+df['%_ZVWKZIEKENHUIS_user'] = df['%_ZVWKZIEKENHUIS_user'].mask( (df['YEAR'] >2020), np.nan)
+df['%_ZVWKOSTENPSYCHO_user'] = df['%_ZVWKOSTENPSYCHO_user'].mask( (df['YEAR'] >2020), np.nan)
+
+# End clean-up
+
+columns = [col for col in df.columns.to_list() if col not in ['WKC', 'GMN', 'Wijknaam', 'YEAR']]
+
+
+drop_var = dcc.Dropdown(
+        columns,
+        'Total_Population',
+        id = 'drop_var_id',
+        clearable=False,
+        searchable=False,
+        className = "custom_select"
+    )
+
+
+drop_municipality = dcc.Dropdown(
+        id = 'drop_municipality',
+        clearable=False, 
+        searchable=False, 
+        # below could be improved as well eventually, by extracting all regions from the data + the special_regions
+        options=[
+            {'label': "Hadoks' area", 'value': "Hadoks' area"},
+            {'label': "'s-gravenhage", 'value': "'s-Gravenhage"},
+            {'label': "Rijswijk", 'value': "Rijswijk"},
+            {'label': 'Leidschendam-Voorburg', 'value': 'Leidschendam-Voorburg'},
+            {'label': 'Wassenaar', 'value': 'Wassenaar'},
+            # {'label': 'Roaz', 'value': 'Roaz'},
+            # {'label': "Haaglanden", 'value': 'Haaglanden'},
+            # {'label': 'Leiden', 'value': 'Leiden'},
+            # {'label': 'Delft', 'value': 'Delft'}
+            ],
+        value="'s-Gravenhage", 
+        className = "custom_select"
+    )
+
+layout = html.Div([
+            html.Div(
+                dbc.Accordion([
+                    dbc.AccordionItem([
+                        html.Div([
+                            html.Div([html.Label('Choose a variable to plot :', id= 'choose_variable', htmlFor= 'drop_var_id'), drop_var], id= 'select_variable'),
+                            html.Div([html.Label('Choose a region to plot:', id='choose_area', htmlFor= 'drop_municipality'), drop_municipality], id = 'select_region'),
+                            html.Div([html.Label('Choose neighbourhoods to plot:', id= 'choose_wijk', htmlFor= 'drop_municipality_spec_id'),
+                                dcc.Dropdown(
+                                    columns,
+                                    id = 'drop_municipality_spec_id',
+                                    clearable=True,
+                                    searchable=True, 
+                                    multi=True,
+                                    className= "custom_select"
+                                )
+                            ], id='select_neighbourhoods'),
+                            html.Div([
+                                "Select year of interest",
+                                dcc.Slider(step=1, id = 'slider_select_year'),
+                                dcc.Dropdown( id = 'drop_select_year', className= "custom_select") #when resolution is small, slider is no longer practical
```

Comment:
I was planning to add some "animation" button on this slider. But I could not. So we can change into drop down later on maybe? But I just thought that it looks nice with a slider. Hahaha